src/pages/Hosting/HostingRouter.js

```diff
@@ -1,52 +1,96 @@
 import React from 'react';
 import { Route, Routes, useNavigate, useLocation } from 'react-router-dom';
+import styled from 'styled-components';
 import HostingFooter from './HostingComponents/HostingFooter';
 import HostingNav from './HostingComponents/HostingNav';
+import StayType from './HostingPages/StayType';
 import SearchForm from './HostingPages/Location/SearchForm';
-import styled from 'styled-components';
-import PrivacyType from './HostingPages/StayType';
+import FloorPlan from './HostingPages/FloorPlan/FloorPlan';
 
-const Customer = () => {
+const HostingRouter = () => {
   const navigate = useNavigate();
   const location = useLocation();
   const newStayInfo = {};
-  let mainQuestion = '';
 
-  //TODO : 추후 switch 문으로 변경예정
-  if (location.pathname === '/hosting') {
-    mainQuestion = '게스트가 머무르게 될 숙소의 종류가 무엇인가요?';
-  } else if (location.pathname === '/hosting/location') {
-    mainQuestion = '숙소 위치는 어디인가요?';
-  }
+  location.pathname === '/hosting' && localStorage.removeItem('stayInfo');
+
+  const addInfo = newStayInfo => {
+    let info = [];
+    if (localStorage.getItem('stayInfo')) {
+      info = JSON.parse(localStorage.getItem('stayInfo'));
+    }
+    info.push(newStayInfo);
+    localStorage.setItem('stayInfo', JSON.stringify(info));
+  };
+
+  const removeInfo = () => {
+    let storageInfo = JSON.parse(localStorage.getItem('stayInfo'));
+    storageInfo.pop();
+    localStorage.setItem('stayInfo', JSON.stringify(storageInfo));
+  };
+
+  const getPageTitle = path => {
+    const title = {
+      '/hosting': '게스트가 머무르게 될 숙소의 종류가 무엇인가요?',
+      '/hosting/location': '숙소 위치는 어디인가요?',
+      '/hosting/floor-plan': '숙소에서 맞이할 최대 인원수를 알려주세요',
+      '/hosting/amenities': '숙소 편의시설 정보를 추가해 주세요',
+    };
+    return title[path];
+  };
 
-  //TODO : 추후 Switch 문으로 변경예정
   const goToNextStep = () => {
-    if (location.pathname === '/hosting') {
-      navigate('/hosting/location');
+    addInfo(newStayInfo);
+    switch (location.pathname) {
+      case '/hosting':
+        navigate('/hosting/location');
+        break;
+      case '/hosting/location':
+        navigate('/hosting/floor-plan');
+        break;
+      case '/hosting/floor-plan':
+        navigate('/hosting/amenities');
+        break;
+      default:
+        navigate('/hosting');
     }
```

Comment:
위에서 객체로 잘 관리해주셨네요! 이부분도 객체로 관리할 수 있습니다:)
```suggestion
const hostingURL = {
      '/hosting/': '/hosting/location',
      '/hosting/location': '/hosting/floor-plan',
      '/hosting/floor-plan': '/hosting/amenities',
    };
    navigate(hostingURL[location.pathname]);
```

Reply:
이 부분 안그래도 객체로 바꾸고 싶었는데 어떻게 바꿔야 할지 몰랐는데 😂 이렇게 바꾸면 되는 거였군요! 다시 한번 객체 활용법 깨우치고 갑니다! 정말 감사합니다 👶🙏